src/cov/Coverage.java

```diff
@@ -512,15 +513,23 @@ public static void initCoverage(Class<?> cls) {
 
 
   /**
-   *
+   * getMemberContaining retrieves the class member that contains the {@link CoverageAtom}
+   * 
    * @param cov
    * @return Returns null if this coverage atom does not belong to a method or
    *         constructor.
+   * @throws IllegalArgumentException if {@link CoverageAtom} argument is null.
```

Comment:
Would it be better to specify that the cov parameter should be non-null (in its `@param` block), rather than specifying precisely what happens if it is?